packages/paste-core/components/radio-button-group/src/StyledRadioButtonLabel.tsx

```diff
@@ -0,0 +1,16 @@
+import {styled, css} from '@twilio-paste/styling-library';
+import {ToggleButtonStyles} from '@twilio-paste/button/src/styles';
+
+export const StyledRadioButtonLabel = styled.label(
+  css({
+    '& > input[type=radio]:checked + span': {
+      ...ToggleButtonStyles._pressed,
+    },
+    '& > input[type=radio]:checked + span:hover': {
+      ...ToggleButtonStyles._pressed_hover,
+    },
+    '& > input[type=radio]:checked:focus + span': {
+      ...ToggleButtonStyles._pressed_focus,
+    },
+  })
```

Comment:
question (blocking): rather than this sibling styling CSS, couldn't it be possible to pass `pressed` directly to the Button on line 112 (https://github.com/twilio-labs/paste/pull/2746/files#diff-a521b250aee50a0f703299853e41f573a04161259b607d498ebe3aec5e408cf4R112) based on the state we already have and are passing to the HiddenRadio??

Reply:
I don't think we can do that because the pressed state uses the `_pressed` pseudo selector which won't happen since this uses `checked` attribute on the hidden radio, not `pressed` attribute on the Button as span.

Reply:
tldr; it's more accessible for it to be styled this way, but we'd like to avoid nested CSS in the future if at all possible.